common/remote/sockfile.cpp

```diff
@@ -3830,6 +4022,78 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
         return false;
     }
 
+    bool associateKeyManager(int handle, IKeyManager *keyManager)
+    {
+        CriticalBlock block(sect);
+        unsigned clientidx;
+        unsigned handleidx;
+        if (!findHandle(handle,clientidx,handleidx))
```

Comment:
I don't know, but one of the reasons I thought (surprised wasn't / had forgotten) that it was using a hash table for lookup..